charts/grafana_dashboard_patches.json

```diff
@@ -0,0 +1,17 @@
+{
+  "etcd": {
+    "uid": "c2f4e12cdf69feb95caa41a5a1b423d9"
+  },
+  "USE Method / Cluster": {
+    "uid": "3e97d1d02672cdd0861f4c97c64f89b2"
+  },
+  "USE Method / Node": {
+    "uid": "fac67cfbe174d3ef53eb473d73d9212f"
+  },
+  "Nodes": {
+    "uid": "fa49a4706d07a042595b664c87fb33ea"
+  },
+  "Prometheus / Overview": {
+    "uid": "prometheus-overview"
```

Comment:
No link to this PR, but wondering if we shouldn't put "explicit" UID for all dashboards

Reply:
Yeah, wondered about the same thing. It could simplify the UI links, at least :)